cmdstanpy/utils.py

```diff
@@ -609,7 +609,7 @@ def scan_optimize_csv(path: str, save_iters: bool = False) -> Dict[str, Any]:
             if save_iters:
                 all_iters[i, :] = [float(x) for x in xs]
             if i == iters - 1:
-                mle = np.array([float(x) for x in xs], dtype=float)
+                mle = np.array([float(x) for x in xs])
```

Comment:
```suggestion
                mle = np.array(xs, dtype=float)
```

Why do we have a list comprehension here?

Reply:
not needed - didn't know np.array could convert strings to float.

Reply:
Yeah, it is not obvious